sites/all/modules/unl_cas/unl_cas.admin.inc

```diff
@@ -162,10 +162,21 @@ function unl_cas_config($form, &$form_state) {
   $form['ldap']['password'] = array(
     '#type' => 'password',
     '#title' => 'Password',
+    '#default_value' => unl_cas_get_setting('ldap_password'),
```

Comment:
dunno, thought i was being clever so that you could update 'seconds' without needing to reenter the password